src/app/components/table-row-option-menu.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+  selector: "app-table-row-option-menu",
```

Comment:
I feel like we could rename this component to `app-menu`. To my knowledge there's nothing that directly couples this to a table row other than the icon? Which we could make customizable similar to https://material.angular.io/components/menu/overview